packages/kbn-esql-validation-autocomplete/src/validation/__tests__/test_suites/validation.command.from.ts

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { METADATA_FIELDS } from '../../../shared/constants';
+import * as helpers from '../helpers';
+
+export const validationFromCommandTestSuite = (setup: helpers.Setup) => {
+  describe('validation', () => {
+    describe('command', () => {
+      describe('FROM <sources> [ METADATA <indices> ]', () => {
+        test('errors on invalid command start', async () => {
+          const { expectErrors } = await setup();
+
+          await expectErrors('f', [
```

Comment:
I like the `expectErrors` dependency injection pattern, but it looks like we have lost the ability to add `.only` and `.skip`. Can we support those?

If so, we should also add `expectErrors.only` to the banned combinations in the [linter config](https://github.com/elastic/kibana/blob/main/packages/kbn-eslint-config/typescript.js#L256-L260) so we don't accidentally skip tests.

Reply:
In this pattern&mdash;really it is just the standard way of writing tests&mdash;we don't need to add `expectErrors.only` or modify the linter configs, because one can just add `.only` to the `test(...` itself.

Like (note `.only`):

```ts
test.only('errors on invalid command start', async () => {
  const { expectErrors } = await setup();
  // ...
});
```

Reply:
Hmm okay. I did like being able to run single test cases easily for debugging. But, I can get there with doing what you say and commenting out some lines.

Reply:
Yes, a test case might have more than one assertion. But since the assertions are grouped with `describe()` and `test()` it is easy to comment some, or often-times you might not even need to comment them (since they are related, they all fail).

Reply:
I'm talking about when I turn on the debugger and step through code. It's annoying to have to step through multiple cases. But, like we've discussed, there are ways around it. I don't dislike this new (old? 😀) pattern.